examples/tutorials/02_cozmo_face/02_cozmo_face_mirror.py

```diff
@@ -56,13 +56,16 @@ def calc_pixel_threshold(image):
     return mean_value
 
 
-def cozmo_face_mirror(robot):
+def cozmo_face_mirror(robot: cozmo.robot.Robot):
```

Comment:
if we're going to use type hints for one function, we should probably use them for all in an example